tutorials/backend/hasura-authentication/tutorial-site/content/integrations/logto.md

```diff
@@ -10,33 +10,43 @@ Logto is an open source identity solution that helps you build the sign-in exper
 
 Main features:
 
-- An OIDC-based identity service
-- Multi-platform user sign-in/up experience with dark mode and SDKs (Web, iOS, and Android)
-- Sign-in/up with dynamic SMS/Email passcode
-- Out-of-box social sign-in integration (GitHub, Google, WeChat, Alipay, etc.)
-- A web UI to control all above (Admin Console)
-- Extendable multi-language support
+
+🧑‍💻 **A frontend-to-backend identity solution**
+
+- OIDC-based authentication and RBAC authorization.
+- Passwordless sign in and much more diverse options, including Email, Phone number, Username, Google, Facebook and other social sign in methods.
+- Beautiful UI components with customizable CSS to fit your business needs.
+
+📦 **Out-of-box infrastructure**
+
+- A ready-to-use management API can serve as your authentication provider, eliminating the need for extra implementation.
+- SDKs that can integrate your apps with Logto quickly, multi-platform and language compatible, tailored to your development environment.
+- Flexible connectors, scalable with community contributions, customizable with SMAL, OAuth, and OIDC protocols.
+
+💻 **Enterprise-ready solutions**
+
+- RBAC to control your resource through scalable role authorization for diverse use cases.
+- User management and audit Logs to understand identity related user info and keep your security on track.
+- We are currently working on SSO, Organizations and MFA! Stay tuned!
```

Comment:
Can we remove this part? The guide talks from Hasura pov and hence the context here could be misunderstood.

Reply:
updated